postalicious/public/styles.css

```diff
@@ -0,0 +1,135 @@
+html, body {
+  height: 97%;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: sans-serif;
+}
+
+.container {
+  height: 100%
+}
+
+.form-section {
+  height: 33%;
+  min-height: 180px;
+  display: flex;
+  flex-direction: column;
+}
+/* Begin of Form Section */
+
+.form-body form {
+  display: flex;
+  flex-direction: column;
+}
+
+fieldset {
+  width: 25%;
+  height: 100%;
+}
+
+.inputButtons {
+  display: flex;
+  align-self: flex-end;
+  margin-top: 10px;
+}
+
+.buildRequest {
+  background-color: #73CE1B;
+}
+
+.buildSend {
+  background-color: #4087DF;
+}
+
+.inputButtons input {
+  margin: 5px;
+  height: 45px;
+  width: 175px;
+  border: none;
+  color: #484B3A;
+  font-size: 1em;
+}
+
+.inputButtons input:active {
+  border: none;
+}
+
+.inputFields {
+  display: flex;
+}
+
+.form-input {
```

Comment:
you're using more than one css classname style here. You have both `CamelCase` and `dash-seperated` and you should pick one.